src/App.tsx

```diff
@@ -1,17 +1,15 @@
-import { Outlet } from 'react-router-dom';
-import { Header } from './components/Header';
+// import { Outlet } from 'react-router-dom';
+import { Header } from 'components/Header';
+import { HomePage } from './pages/HomePage';
 import './index.scss';
 
 export const App = () => {
   return (
     <html lang="en" className="page">
+      <Header />
+
       <body className="page__body">
```

Comment:
why do you still have html and body??

you should have your app like this 
```suggestion
    export const App = () => {
  return (
    <div className="page">
      <Header />
      <main className="page__body">
        <Outlet />
      </main>
      <Footer />
    </div>
  );
};
```